doc/api/http2.md

```diff
@@ -1296,10 +1326,10 @@ Note that using the same file descriptor concurrently for multiple streams
 is not supported and may result in data loss. Re-using a file descriptor
 after a stream has finished is supported.
 
-When set, the `options.getTrailers()` function is called immediately after
-queuing the last chunk of payload data to be sent. The callback is passed a
-single object (with a `null` prototype) that the listener may use to specify
-the trailing header fields to send to the peer.
+When the `options.hasTrailers` option is set, the `'trailers-ready'` event will
+be emitted immediately after queuing the last chunk of payload data to be sent.
+The `http2stream.trailers()` method can then be used to sent trailing header
+fields to the peer.
```

Comment:
if `hasTrailers: true`, then the user __must__ call `.trailers()` to close the writable side of the stream. This is not being stressed enough in the sentence. As far as I understand, if `.trailers()`  is not called,  there will be a memory leak.
This seems a dangerous API.

Reply:
All of that should definitely be documented.